From e767f892e7736008b60f85d2c2690d660661814b Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@lantiq.com>
Date: Thu, 13 Aug 2015 15:03:11 +0200
Subject: [PATCH] resource-encapsulation: fix build without curl

The hue plugin needs curl and it should only be build when libcurl is available.
Without this patch, the build fails without curl installed, this patch fixes the problem and makes the build worm without curl and without the hue stuff.

Without this patch I am getting this error message:
scons: ***
File "/service/third_party_libs.scons", line 50, in ?

Change-Id: I5b4da555ff84b9b605cc6c119990d60ff670bd0d
Signed-off-by: Hauke Mehrtens <hauke.mehrtens@lantiq.com>
---
 .../src/resourceContainer/SConscript               |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/service/resource-encapsulation/src/resourceContainer/SConscript
+++ b/service/resource-encapsulation/src/resourceContainer/SConscript
@@ -177,7 +177,7 @@ else:
 	
 	HueBundle = hue_resource_bundle_env.SharedLibrary('HueBundle', hue_resource_bundle_src)
 	hue_resource_bundle_env.InstallTarget(HueBundle, 'libHueBundle')
-	lib_env = conf2.Finish()
+lib_env = conf2.Finish()
 
 ######################################################################
 # build resource container unit tests
